### PR TITLE
Erlang: bump version, recipe cleanup

### PR DESCRIPTION
**Pull request by miqlas**
It makes https://github.com/haikuports/haikuports/pull/1223 outdated.

### CODE REVIEW

File: dev-lang/erlang/erlang-19.3.recipe [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	cmd:run_erl = $portVersion
+	cmd:to_erl = $portVersion
+	cmd:typer = $portVersion
+	"
+REQUIRES="
+	haiku
+	cmd:libtool
+	cmd:gcc
+	cmd:perl
+	cmd:wx_config
```
Comment (korli) [inline dev-lang/erlang/erlang-19.3.recipe]: Wx_config, really?

**Comment by miqlas** [comment]
why not?

**Comment by korli** [comment]
Well, Erlang didn't require wx_config until this change. You should definitely explain things, not hide every change for us to find out.

**Comment by miqlas** [comment]
Wings3D requires it.

**Comment by return** [comment]
@miqlas Erlang 19.3 doesn't seem compile on x86_gcc2. You should mark x86_gcc2 as broken until there is a patch that fixes this.

**Comment by miqlas** [comment]
return, thanks, disabled.

**Comment by korli** [comment]
REQUIRES should probably have some runtime dependency to wxqt, not related to wxqt_devel.

**Pull request merged**